glancesync/glancesync_ami.py

```diff
@@ -42,19 +42,18 @@
 _logger = logging.getLogger('glancesync')
 
 
-def get_master_region_dict(image_list):
-    """Convert the image list to a dictionary indexed by name and also
-    replace at parameters kernel_id and ramdisk_id the UUID with the name of
+def clean_ami_ids(image_dict):
+    """replace at parameters kernel_id and ramdisk_id the UUID with the name of
     the image.
 
-    :param image_list: the list of images on the master region
-    :return: a dictionary indexed by name
+    :param image_dict: a dictionary of master images
+    :return: nothing
```

Comment:
It does what the docstring says. It does not need to return/print nothing, because if receives a dictionary and iterates over the dictionary changing the elements of the dictionary.